source/documentdb-support.txt

```diff
@@ -0,0 +1,84 @@
+.. _documentdb-support:
+
+===============================
+Amazon DocumentDB Compatibility
+===============================
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta:: 
+   :keywords: AWS, support
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+.. figure:: /figures/documentdb.png
+   :alt: Amazon DocumentDB logo
+
+`Amazon DocumentDB (with MongoDB compatibility) <https://docs.aws.amazon.com/documentdb/latest/developerguide/what-is.html>`__
+is a fast, reliable, and fully managed database service. Amazon DocumentDB makes it easy
+to set up, operate, and scale MongoDB-compatible databases in the cloud.
+With Amazon DocumentDB, you can run the same application code and use the same drivers
+and tools that you use with MongoDB.
+
+Version Information
+-------------------
+
+Amazon DocumentDB supports MongoDB v4.0 and MongoDB v5.0. These versions lack
+support for the newer features and functionality in the versions of MongoDB available
+on `MongoDB Atlas. <https://www.mongodb.com/docs/atlas/>`__
```

Comment:
Suggestion: Since this is a marketing-type doc, I would avoid using words with negative associations like "lack".

```suggestion
Amazon DocumentDB supports MongoDB v4.0 and MongoDB v5.0. These legacy versions do not 
support some of the newer features available on `MongoDB Atlas. <https://www.mongodb.com/docs/atlas/>`__, 
but additional capabilities and management tools are available with Amazon DocumentDB.
```

Note: What I wrote is a guess, but the point is to focus on benefits, not limitations.

Reply:
Good point.